src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceCommand.cs

```diff
@@ -0,0 +1,220 @@
+    {
+        /// <summary>
+        /// The fixed command format.
+        /// </summary>
+        Fixed = 0,
+        /// <summary>
+        /// The fixed and variable fixed command format.
+        /// </summary>
+        FixedAndVFixed = 1,
+        /// <summary>
+        /// The variable fixed and fixed command format.
+        /// </summary>
+        VFixedAndFixed = 2,
+        /// <summary>
+        /// The fixed and non-fixed command format.
+        /// </summary>
+        FixedAndNonFixed = 3,
+        /// <summary>
+        /// The non-fixed and fixed command format.
+        /// </summary>
+        NonFixedAndFixed = 4,
+        /// <summary>
+        /// The action command
+        /// </summary>
+        Action = 5,
+        /// <summary>
+        /// The partial matched command
+        /// </summary>
+        Partial = 6,
+        /// <summary>
+        /// Undefined
+        /// </summary>
+        Undefined = -1
+    };
+
+    /// <summary>
+    /// This class represents a voice command.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class VoiceCommand
+    {
+        internal SafeCommandHandle _handle;
+
+        /// <summary>
+        /// The public constructor.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <exception cref="InvalidOperationException">
+        /// This can occur due to the following reasons:
+        /// 1) This exception can be due to an invalid state.
+        /// 2) This exception can be due to out of memory.
+        /// </exception>
+        /// <exception cref="UnauthorizedAccessException">This exception can be due to permission denied.</exception>
+        /// <exception cref="NotSupportedException">This exception can be due to not supported.</exception>
+        public VoiceCommand()
+        {
+            SafeCommandHandle handle;
+            ErrorCode error = VcCmdCreate(out handle);
+            if (error != ErrorCode.None)
+            {
+                Log.Error(LogTag, "Create Failed with error " + error);
+                throw ExceptionFactory.CreateException(error);
+            }
+            _handle = handle;
+        }
+
+        internal VoiceCommand(SafeCommandHandle handle)
+        {
+            _handle = handle;
+        }
+
+        /// <summary>
+        /// Gets the unfixed command.
+        /// This property should be used for commands which have non-fixed format.
+        /// An empty string will be returned in case of some internal error.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public string UnfixedCommand
+        {
+            get
+            {
+                string unfixedCommand;
+                ErrorCode error = VcCmdGetUnfixedCommand(_handle, out unfixedCommand);
+                if (error != ErrorCode.None)
+                {
+                    Log.Error(LogTag, "UnfixedCommand Failed with error " + error);
+                    return "";
```

Comment:
```suggestion
                    return string.Empty;
```

Reply:
You do not need to returin empty string becuase a default string value is empty string
```suggestion
```

Reply:
Fixed.